data-raw/prepare_data_for_model.R

```diff
@@ -317,6 +317,33 @@ degree_days_sr <- list(biop_2008_2009 = degree_days_2008_2009_sr,
 
 usethis::use_data(degree_days_sr, overwrite = TRUE)
 
+# degree days for above dam that max out at 17 degrees
+# TODO: is 17 the correct threshold value. Double check. Explore how the threshold affects output
```

Comment:
This says 17 but looks like below you have it actually at 13. Please update the comment (I will check with tech team on Monday about the temperature threshold)